rcl/Doxyfile

```diff
@@ -0,0 +1,17 @@
+# All settings not listed here will use the Doxygen default values.
+
+PROJECT_NAME           = "rcl"
+PROJECT_NUMBER         = beta1
```

Comment:
I am not sure if embedding "beta1" here is s good idea. I looks like that would be yet another location in many repos which needs to be updated.